### ISSUE TITLE
[YSQL][Unit Test] Test TestDeleteCDCStreamWithBootstrap failing with yb-admin termination

### ISSUE THREAD

**Issue by rjalan-yb** [yugabyte/yugabyte-db#18731]
Jira Link: [DB-7627](https://yugabyte.atlassian.net/browse/DB-7627)
### Description

Unit Test [XClusterAdminCliTest.TestDeleteCDCStreamWithBootstrap](https://detective-gcp.dev.yugabyte.com/stability/test?analyze_trends=true&branch=master&build_type=all&class=XClusterAdminCliTest&fail_tag=all&name=TestDeleteCDCStreamWithBootstrap&platform=linux) started failing with yb-admin process exit error after this commit: [d4fd03cb](https://github.com/yugabyte/yugabyte-db/commit/d4fd03cb6c16e2687d9a5e5a3b0cf67528c2acd8)

Error:
```
../../src/yb/tools/yb-admin-xcluster-test.cc:786
Failed
Bad status: Runtime error (yb/util/subprocess.cc:631): Subprocess '/nfusr/alma8-gcp-cloud/jenkins-worker-ndnh3g/jenkins/jenkins-github-yugabyte-db-alma8-master-gcc11-fastdebug-3408/yugabyte-db/build/fastdebug-gcc11-dynamic-ninja/tests-tools/../bin/yb-admin' terminated with non-zero exit status 256
```

Complete Logs: https://jenkins.dev.yugabyte.com/job/github-yugabyte-db-alma8-master-gcc11-fastdebug/3408/artifact/build/fastdebug-gcc11-dynamic-ninja/yb-test-logs/tests-tools__yb-admin-xcluster-test/XClusterAdminCliTest_TestDeleteCDCStreamWithBootstrap.log

### Warning: Please confirm that this issue does not contain any sensitive information

- [X] I confirm this issue does not contain any sensitive information.

[DB-7627]: https://yugabyte.atlassian.net/browse/DB-7627?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

**Comment by rjalan-yb** [comment]
Same error is seen for test [XClusterAdminCliTest.TestListCdcStreamsWithBootstrappedStreams](https://detective-gcp.dev.yugabyte.com/stability/test?analyze_trends=true&branch=master&build_type=all&class=XClusterAdminCliTest&fail_tag=all&name=TestListCdcStreamsWithBootstrappedStreams&platform=linux) around the same commit.

**Comment by hulien22** [comment]
@rjalan-yb Its a race condition when running bootstrap immediately on newly created tables, where we can then have tablets that aren't fully initialized on the cdc-service tserver.

Mostly a test issue, working on a fix that will fix all of the xcluster tests that run bootstraps

**Comment by hulien22** [comment]
Fixed by 1394668963a17330478dfba53dbdbabc037bbe23
